From 21caf62b4f60cce47a5d3b8c939cc7b177904e00 Mon Sep 17 00:00:00 2001
From: Jonas Herzig <me@johni0702.de>
Date: Mon, 7 Aug 2017 19:52:13 +0200
Subject: [PATCH] Configure gradle to build reproducible archives

---
 fernflower.gradle | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/fernflower.gradle b/fernflower.gradle
index 78a2be1..468733e 100644
--- a/fernflower.gradle
+++ b/fernflower.gradle
@@ -12,4 +12,11 @@ compileJava {
 }
 
+tasks.withType(AbstractArchiveTask) {
+    preserveFileTimestamps = false
+    reproducibleFileOrder = true
+    dirMode = 0775
+    fileMode = 0664
+}
+
 sourceSets {
     main.java.srcDirs = ['src']
-- 
2.5.5

