From e9fa8966b42a3d9f35b7aa0153a1466bdddda7c2 Mon Sep 17 00:00:00 2001
From: Jonas Herzig <me@johni0702.de>
Date: Sat, 28 Oct 2017 19:15:34 +0200
Subject: [PATCH] Replace plugins block with buildscript

---
 build.gradle | 13 +++++++++++--
 1 file changed, 11 insertions(+), 2 deletions(-)

diff --git a/build.gradle b/build.gradle
index da78c2f..faffc80 100644
--- a/build.gradle
+++ b/build.gradle
@@ -1,7 +1,16 @@
-plugins {
-    id 'net.minecrell.gitpatcher' version '0.9.0'
+buildscript {
+  repositories {
+    maven {
+      url "https://plugins.gradle.org/m2/"
+    }
+  }
+  dependencies {
+    classpath "gradle.plugin.net.minecrell:gitpatcher:0.9.0"
+  }
 }
 
+apply plugin: "net.minecrell.gitpatcher"
+
 patches {
     submodule = 'FernFlower'
     target = file('ForgeFlower')
-- 
2.9.5

