From 798d79d8c6b5c4accda35bd4517621b19bf954a2 Mon Sep 17 00:00:00 2001
From: Jonas Herzig <me@johni0702.de>
Date: Tue, 29 Aug 2017 13:47:11 +0200
Subject: [PATCH] Replace plugin DSL with old buildscript

---
 build.gradle | 13 +++++++++++--
 1 file changed, 11 insertions(+), 2 deletions(-)

diff --git a/build.gradle b/build.gradle
index da78c2f..4384886 100644
--- a/build.gradle
+++ b/build.gradle
@@ -1,7 +1,16 @@
-plugins {
-    id 'net.minecrell.gitpatcher' version '0.9.0'
+buildscript {
+    repositories {
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
+    }
+    dependencies {
+        classpath "gradle.plugin.net.minecrell:gitpatcher:0.9.0"
+    }
 }
 
+apply plugin: "net.minecrell.gitpatcher"
+
 patches {
     submodule = 'FernFlower'
     target = file('ForgeFlower')
-- 
2.5.5

