From ba15c05e6afa05948c30b53b49baaff9211e9abe Mon Sep 17 00:00:00 2001
From: Jonas Herzig <me@johni0702.de>
Date: Mon, 28 Aug 2017 21:17:16 +0200
Subject: [PATCH 2/2] Make sure shadowJar is run before build task

---
 build.gradle | 1 +
 1 file changed, 1 insertion(+)

diff --git a/build.gradle b/build.gradle
index dc84f48..d99f39b 100644
--- a/build.gradle
+++ b/build.gradle
@@ -101,6 +101,7 @@ shadowJar {
     append('META-INF/NOTICE.txt')
     from(tasks.findByName('preshadowJar').outputs.files.collect {it.isDirectory() ? it : zipTree(it)})
 }
+build.dependsOn shadowJar
 
 eclipse.project {
         name = 'ReplayStudio'
-- 
2.5.5

