From e60dd702c247692529a7cece94a475650623482e Mon Sep 17 00:00:00 2001
From: Jonas Herzig <me@johni0702.de>
Date: Tue, 8 Aug 2017 19:05:14 +0200
Subject: [PATCH 1/7] Configure gradle to build reproducible archives

---
 build.gradle | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/build.gradle b/build.gradle
index 4f0aa28..e0c46ea 100644
--- a/build.gradle
+++ b/build.gradle
@@ -170,6 +170,13 @@ processResources {
    from 'LICENSE.txt'
 }
 
+tasks.withType(AbstractArchiveTask) {
+    preserveFileTimestamps = false
+    reproducibleFileOrder = true
+    dirMode = 0775
+    fileMode = 0664
+}
+
 // License header formatting
 license {
     ext {
-- 
2.5.5

