From 0f250141ba6515dbeffeb0722bf29976850bb323 Mon Sep 17 00:00:00 2001
From: Jonas Herzig <me@johni0702.de>
Date: Tue, 8 Aug 2017 19:13:17 +0200
Subject: [PATCH 3/7] Fix for Gradle 3

---
 build.gradle | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build.gradle b/build.gradle
index 165dbe4..0f189ec 100644
--- a/build.gradle
+++ b/build.gradle
@@ -92,7 +92,7 @@ dependencies {
 
 // Task to read ASM library and rename packages
 task renamedASM {
-    outputs.file files(configurations.asm.files.findAll{!it.name.contains("sources")}.collect { file ->
+    outputs.files files(configurations.asm.files.findAll{!it.name.contains("sources")}.collect { file ->
         new File(project.repositories['renamed-packages'].dirs[0], "mixin-" + file.name).path
     })
     doLast {
-- 
2.5.5

