From d06cec4bfa5981965faf2dad510d14e000d61ade Mon Sep 17 00:00:00 2001
From: Jonas Herzig <me@johni0702.de>
Date: Tue, 8 Aug 2017 18:53:57 +0200
Subject: [PATCH 3/4] Add witness plugin

---
 build.gradle | 21 +++++++++++++++++++++
 1 file changed, 21 insertions(+)

diff --git a/build.gradle b/build.gradle
index 1b06411..a0d350b 100644
--- a/build.gradle
+++ b/build.gradle
@@ -1,4 +1,11 @@
+buildscript {
+    dependencies {
+        classpath files('../gradle-witness.jar')
+    }
+}
+
 apply plugin: 'groovy'
+apply plugin: 'witness'
 
 repositories {
     mavenLocal()
@@ -18,6 +25,20 @@ dependencies {
     compile 'org.codehaus.groovy:groovy-backports-compat23:2.4.4'
 }
 
+dependencyVerification {
+    verify = [
+        'org.jdom:jdom2:2.0.5:32ed093f39c46a8d99f801240069478eac93b35f07351c001f2cef4330e97b4a',
+        'org.ow2.asm:asm:5.0.3:71c4f78e437b8fdcd9cc0dfd2abea8c089eb677005a6a5cff320206cc52b46cc',
+        'org.ow2.asm:asm-commons:5.0.3:18c1e092230233c9d29e46f21943d769bdb48130cc279e4b0e663f423948c2da',
+        'commons-io:commons-io:2.4:cc6a41dc3eaacc9e440a6bd0d2890b20d36b4ee408fe2d67122f328bb6e01581',
+        'org.apache.ant:ant:1.9.4:649ae0730251de07b8913f49286d46bba7b92d47c5f332610aa426c4f02161d8',
+        'org.codehaus.plexus:plexus-utils:2.0.6:8b909f4ca9788647942f883d4e559bcc642123f7c6bcd3846983a2e465469c33',
+        'org.codehaus.groovy:groovy-backports-compat23:2.4.4:f07694998acb204782c72cc8067b237a04097a2170f07d440614883f14a67630',
+        'org.ow2.asm:asm-tree:5.0.3:347a7a9400f9964e87c91d3980e48eebdc8d024bc3b36f7f22189c662853a51c',
+        'org.apache.ant:ant-launcher:1.9.4:7bccea20b41801ca17bcbc909a78c835d0f443f12d639c77bd6ae3d05861608d',
+    ]
+}
+
 jar {
     from rootProject.file('LICENSE')
     from rootProject.file('NOTICE')
-- 
2.5.5

