From cfca21accce1fb3fbd32b81fb79f0363dd7b61f4 Mon Sep 17 00:00:00 2001
From: Jonas Herzig <me@johni0702.de>
Date: Tue, 8 Aug 2017 19:59:34 +0200
Subject: [PATCH 4/4] Manually shade deps

---
 build.gradle | 20 +++++++++++++-------
 1 file changed, 13 insertions(+), 7 deletions(-)

diff --git a/build.gradle b/build.gradle
index a0d350b..b12a46d 100644
--- a/build.gradle
+++ b/build.gradle
@@ -12,17 +12,22 @@ repositories {
     jcenter()
 }
 
+configurations {
+    shade
+    compile.extendsFrom shade
+}
+
 dependencies {
 
     compile localGroovy()
     compile gradleApi()
-    compile 'org.jdom:jdom2:2.0.5'
-    compile 'org.ow2.asm:asm:5.0.3'
-    compile 'org.ow2.asm:asm-commons:5.0.3'
-    compile 'commons-io:commons-io:2.4'
-    compile 'org.apache.ant:ant:1.9.4'
-    compile 'org.codehaus.plexus:plexus-utils:2.0.6'
-    compile 'org.codehaus.groovy:groovy-backports-compat23:2.4.4'
+    shade 'org.jdom:jdom2:2.0.5'
+    shade 'org.ow2.asm:asm:5.0.3'
+    shade 'org.ow2.asm:asm-commons:5.0.3'
+    shade 'commons-io:commons-io:2.4'
+    shade 'org.apache.ant:ant:1.9.4'
+    shade 'org.codehaus.plexus:plexus-utils:2.0.6'
+    shade 'org.codehaus.groovy:groovy-backports-compat23:2.4.4'
 }
 
 dependencyVerification {
@@ -42,6 +47,7 @@ dependencyVerification {
 jar {
     from rootProject.file('LICENSE')
     from rootProject.file('NOTICE')
+    from (files(configurations.shade).collect {it.isDirectory() ? filesTree(it) : zipTree(it)})
 }
 
 tasks.withType(AbstractArchiveTask) {
-- 
2.5.5

