From 9baa28ace59f2036e93fff379438c2553f02b6a8 Mon Sep 17 00:00:00 2001
From: Jonas Herzig <me@johni0702.de>
Date: Mon, 7 Aug 2017 20:41:46 +0200
Subject: [PATCH 3/5] Add witness plugin

---
 build.gradle | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/build.gradle b/build.gradle
index bfa3914..bee78ee 100755
--- a/build.gradle
+++ b/build.gradle
@@ -4,11 +4,13 @@ buildscript {
     dependencies {
         classpath files('gradle/reprod/deps/forgegradle.jar')
         classpath files('gradle/reprod/deps/mixingradle.jar')
+        classpath files('gradle/reprod/deps/gradle-witness.jar')
     }
 }
 
 apply plugin: 'net.minecraftforge.gradle.forge'
 apply plugin: 'org.spongepowered.mixin'
+apply plugin: 'witness'
 
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
@@ -56,6 +58,10 @@ dependencies {
     testCompile 'junit:junit:4.11'
 }
 
+dependencyVerification {
+    includedConfigurations += [configurations.shade, configurations.testCompile]
+}
+
 tasks['idea'].dependsOn ':ReplayStudio:preshadowJar'
 
 jar {
-- 
2.5.5

