From 29d6f245fb91762580540180d4ec6ca4abb18322 Mon Sep 17 00:00:00 2001
From: Jonas Herzig <me@johni0702.de>
Date: Sat, 26 Aug 2017 17:35:04 +0200
Subject: [PATCH 2/2] Remove non-essential parts of build.gradle

---
 build.gradle | 103 -----------------------------------------------------------
 1 file changed, 103 deletions(-)

diff --git a/build.gradle b/build.gradle
index fda0b61..eefeafe 100644
--- a/build.gradle
+++ b/build.gradle
@@ -1,16 +1,3 @@
-buildscript {
-    repositories {
-        mavenCentral()
-        maven {
-            name = "forge"
-            url = "http://files.minecraftforge.net/maven"
-        }
-        maven {
-            name = "eclipse"
-            url = "https://repo.eclipse.org/content/groups/eclipse/"
-        }
-    }
-}
 apply plugin: 'java'
 apply plugin: 'idea'
 apply plugin: 'eclipse'
@@ -22,10 +9,6 @@ configurations {
 targetCompatibility = '1.6'
 sourceCompatibility = '1.6'
 
-configurations {
-    deployerJars
-}
-
 dependencies {
     testCompile "junit:junit:4.12"
     
@@ -45,9 +28,6 @@ dependencies {
     
     // Potential Eclipse AST replacement (not used yet)
     //compile 'fr.inria.gforge.spoon:spoon-core:5.1.0'
-    
-    // maven deployer jar
-    deployerJars "org.apache.maven.wagon:wagon-ssh:2.2"
 }
 
 repositories {
@@ -73,86 +53,3 @@ jar {
     exclude 'data/**'
 }
 
-task javadocJar(type: Jar, dependsOn: javadoc) {
-    classifier = 'javadoc'
-    from 'build/docs/javadoc'
-}
-
-task sourcesJar(type: Jar) {
-    from sourceSets.main.allSource
-    classifier = 'sources'
-}
-
-artifacts {
-    archives jar
-    archives javadocJar
-    archives sourcesJar
-}
-
-uploadArchives {
-    repositories.mavenDeployer {
-
-        dependsOn 'build'
-
-        if (project.hasProperty('forgeMavenPass'))
-        {
-            repository(url: "http://files.minecraftforge.net/maven/manage/upload") {
-                authentication(userName: "forge", password: project.getProperty('forgeMavenPass'))
-            }
-        }
-        else
-        {
-            // local repo folder. Might wanna juset use  gradle install   if you wanans end it to maven-local
-            repository(url: 'file://localhost/' + project.file('repo').getAbsolutePath())
-        }
-
-        pom {
-            groupId = project.group
-            version = project.version
-            artifactId = project.archivesBaseName
-            project {
-                name project.archivesBaseName
-                packaging 'jar'
-                description 'Srg2Source library for ForgeGradle'
-                url 'https://github.com/MinecraftForge/Srg2Source'
-
-                scm {
-                    url 'https://github.com/MinecraftForge/Srg2Source'
-                    connection 'scm:git:git://github.com/MinecraftForge/Srg2Source.git'
-                    developerConnection 'scm:git:git@github.com:MinecraftForge/Srg2Source.git'
-                }
-
-                issueManagement {
-                    system 'github'
-                    url 'https://github.com/MinecraftForge/Srg2Source/issues'
-                }
-
-                licenses {
-                    license {
-                        name 'The BSD 3-Clause License'
-                        url 'http://opensource.org/licenses/BSD-3-Clause'
-                        distribution 'repo'
-                    }
-                }
-
-                developers {
-                    developer {
-                        id 'agaricusb'
-                        name 'Agaricusb'
-                        roles { role 'developer' }
-                    }
-                    developer {
-                        id 'LexManos'
-                        name 'Lex Manos'
-                        roles { role 'developer' }
-                    }
-                    developer {
-                        id 'AbrarSyed'
-                        name 'Abrar Syed'
-                        roles { role 'developer' }
-                    }
-                }
-            }
-        }
-    }
-}
-- 
2.5.5

