From 7266eb65165cfe18ea21a3c183da4efc37ff54cf Mon Sep 17 00:00:00 2001
From: Jonas Herzig <me@johni0702.de>
Date: Sat, 26 Aug 2017 17:46:46 +0200
Subject: [PATCH] Manually shade deps

---
 build.gradle | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/build.gradle b/build.gradle
index eefeafe..368f6be 100644
--- a/build.gradle
+++ b/build.gradle
@@ -61,6 +61,8 @@ jar {
 }
 
 jar {
+    duplicatesStrategy = DuplicatesStrategy.EXCLUDE
+    from (files(configurations.compile).collect {it.isDirectory() ? filesTree(it) : zipTree(it)})
     exclude 'data/**'
 }
 
-- 
2.5.5

