From 21caf62b4f60cce47a5d3b8c939cc7b177904e00 Mon Sep 17 00:00:00 2001
From: Jonas Herzig <me@johni0702.de>
Date: Mon, 7 Aug 2017 19:52:13 +0200
Subject: [PATCH 1/5] Configure gradle to build reproducible archives

---
 build.gradle | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/build.gradle b/build.gradle
index 78a2be1..468733e 100644
--- a/build.gradle
+++ b/build.gradle
@@ -138,6 +138,13 @@ jar {
     }
 }
 
+tasks.withType(AbstractArchiveTask) {
+    preserveFileTimestamps = false
+    reproducibleFileOrder = true
+    dirMode = 0775
+    fileMode = 0664
+}
+
 javadoc {
 	classpath += configurations.compileOnly
 
-- 
2.5.5

