From a4a52f38bd660caa61531ebecd2d11f764eb3691 Mon Sep 17 00:00:00 2001
From: Jonas Herzig <me@johni0702.de>
Date: Mon, 7 Aug 2017 19:52:46 +0200
Subject: [PATCH 3/5] Remove all buildscript dependencies

---
 build.gradle | 130 -----------------------------------------------------------
 1 file changed, 130 deletions(-)

diff --git a/build.gradle b/build.gradle
index 91ea99a..8959f8f 100644
--- a/build.gradle
+++ b/build.gradle
@@ -1,21 +1,7 @@
-buildscript {
-    repositories {
-        maven {
-            url "https://plugins.gradle.org/m2/"
-        }
-    }
-    dependencies {
-        classpath "com.gradle.publish:plugin-publish-plugin:0.9.1"
-        classpath 'nl.javadude.gradle.plugins:license-gradle-plugin:0.11.0'
-    }
-}
-
 apply plugin: 'java'
 apply plugin: 'idea'
 apply plugin: 'eclipse'
 apply plugin: 'maven'
-apply plugin: "com.gradle.plugin-publish"
-apply plugin: 'license'
 
 group = 'net.minecraftforge.gradle'
 version = '2.3-SNAPSHOT'
@@ -188,122 +174,6 @@ test {
         exclude "**/ExtensionMcpMappingTest*"
 }
 
-license {
-    ext {
-        description = 'A Gradle plugin for the creation of Minecraft mods and MinecraftForge plugins.'
-        year = '2013'
-        fullname = 'Minecraft Forge'
-    }
-    header rootProject.file('HEADER')
-    include '**net/minecraftforge/gradle/**/*.java'
-    excludes ([
-        '**net/minecraftforge/gradle/util/ZipFileTree.java',
-        '**net/minecraftforge/gradle/util/json/version/*',
-        '**net/minecraftforge/gradle/util/patching/Base64.java',
-        '**net/minecraftforge/gradle/util/patching/ContextualPatch.java'
-    ])
-    ignoreFailures false
-    strictCheck true
-    mapping {
-        java = 'SLASHSTAR_STYLE'
-    }
-}
-
-pluginBundle {
-    website = 'http://www.gradle.org/'
-    vcsUrl = 'https://github.com/MinecraftForge/ForgeGradle'
-    description = 'Gradle plugin for all Minecraft mod development needs'
-    tags = ['forge', 'minecraft', 'minecraftforge', 'sponge', 'mcp']
-
-    plugins {
-        patcher {
-            id = 'net.minecraftforge.gradle.patcher'
-            displayName = 'Minicraft Patcher Plugin'
-        }
-        tweakerClient {
-            id = 'net.minecraftforge.gradle.tweaker-client'
-            displayName = 'Mincraft Client Tweaker Plugin'
-        }
-        tweakerServer {
-            id = 'net.minecraftforge.gradle.tweaker-server'
-            displayName = 'Mincraft Server Tweaker Plugin'
-        }
-        forge {
-            id = 'net.minecraftforge.gradle.forge'
-            displayName = 'MincraftForge Mod Development Plugin'
-        }
-
-        launch4j {
-            id = 'net.minecraftforge.gradle.launch4j'
-            displayName = 'Specialized Launch4J Gradle Plugin'
-        }
-    }
-}
-
-uploadArchives {
-    repositories.mavenDeployer {
-
-        dependsOn 'build'
-
-        if (project.hasProperty('forgeMavenPass'))
-        {
-            repository(url: "http://files.minecraftforge.net/maven/manage/upload") {
-                authentication(userName: "forge", password: project.getProperty('forgeMavenPass'))
-            }
-        }
-        else
-        {
-            // local repo folder. Might wanna juset use  gradle install   if you wanans end it to maven-local
-            repository(url: 'file://localhost/' + project.file('repo').getAbsolutePath())
-        }
-
-
-        pom {
-            groupId = project.group
-            version = project.version
-            artifactId = project.archivesBaseName
-            project {
-                name project.archivesBaseName
-                packaging 'jar'
-                description 'Gradle plugin for Forge'
-                url 'https://github.com/MinecraftForge/ForgeGradle'
-
-                scm {
-                    url 'https://github.com/MinecraftForge/ForgeGradle'
-                    connection 'scm:git:git://github.com/MinecraftForge/ForgeGradle.git'
-                    developerConnection 'scm:git:git@github.com:MinecraftForge/ForgeGradle.git'
-                }
-
-                issueManagement {
-                    system 'github'
-                    url 'https://github.com/MinecraftForge/ForgeGradle/issues'
-                }
-
-                licenses {
-                    license {
-                        name 'Lesser GNU Public License, Version 2.1'
-                        url 'https://www.gnu.org/licenses/lgpl-2.1.html'
-                        distribution 'repo'
-                    }
-                }
-
-                developers {
-                    developer {
-                        id 'AbrarSyed'
-                        name 'Abrar Syed'
-                        roles { role 'developer' }
-                    }
-                    developer {
-                        id 'LexManos'
-                        name 'Lex Manos'
-                        roles { role 'developer' }
-                    }
-                }
-            }
-        }
-    }
-}
-
 // write out version so its convenient for doc deployment
 file('build').mkdirs()
 file('build/version.txt').text = version;
-- 
2.5.5

