From bf77ac2f526b90bd0c04a2aca7e8d9a1bc90d014 Mon Sep 17 00:00:00 2001
From: Jonas Herzig <me@johni0702.de>
Date: Thu, 8 Mar 2018 17:17:32 +0100
Subject: [PATCH] Fix build for newer gradle versions


diff --git a/build.gradle b/build.gradle
index 5e61544..737ce9e 100644
--- a/build.gradle
+++ b/build.gradle
@@ -114,7 +114,7 @@ artifacts {
 }
 
 //These should add a compile time only dependancies to eclipse, ide, and javac, but NOT to the generated POM on maven.
-sourceSets.main.compileClasspath += [ configurations.compileOnly ]
+sourceSets.main.compileClasspath += configurations.compileOnly
 idea.module.scopes.PROVIDED.plus += [ configurations.compileOnly ]
 eclipse.classpath.plusConfigurations += [ configurations.compileOnly ]
 
-- 
2.13.6

