From f9056380f0e5955817df282c2ac1c8e60c9fedc2 Mon Sep 17 00:00:00 2001
From: Jonas Herzig <me@johni0702.de>
Date: Thu, 8 Mar 2018 17:21:53 +0100
Subject: [PATCH] Fix compilation with ECJ


diff --git a/build.gradle b/build.gradle
index 737ce9e..75c2b62 100644
--- a/build.gradle
+++ b/build.gradle
@@ -90,7 +90,7 @@ dependencies {
 
 compileJava {
 	options.deprecation = true
-	options.compilerArgs += ["-Werror"]
+	//options.compilerArgs += ["-Werror"]
 	//options.compilerArgs += ["-Werror", "-Xlint:unchecked"]
 }
 
-- 
2.13.6

