From 58e8ce507b599b45479aa0d6d823a4e285aecb58 Mon Sep 17 00:00:00 2001
From: Jonas Herzig <me@johni0702.de>
Date: Thu, 8 Mar 2018 19:38:05 +0100
Subject: [PATCH] Remove redundant -sourcepath compiler argument


diff --git a/src/main/java/net/minecraftforge/gradle/user/UserBasePlugin.java b/src/main/java/net/minecraftforge/gradle/user/UserBasePlugin.java
index 4d39d52..f2995e9 100644
--- a/src/main/java/net/minecraftforge/gradle/user/UserBasePlugin.java
+++ b/src/main/java/net/minecraftforge/gradle/user/UserBasePlugin.java
@@ -354,17 +354,6 @@ public abstract class UserBasePlugin<T extends UserExtension> extends BasePlugin
 
         project.getConfigurations().getByName("apiCompile").extendsFrom(project.getConfigurations().getByName("compile"));
         project.getConfigurations().getByName("testCompile").extendsFrom(project.getConfigurations().getByName("apiCompile"));
-        
-        // set compile not to take from libs
-        JavaCompile compileTask = ((JavaCompile)project.getTasks().getByName(main.getCompileJavaTaskName()));
-        List<String> args = compileTask.getOptions().getCompilerArgs();
-        if (args == null || args.isEmpty())
-        {
-            args = Lists.newArrayList();
-        }
-        args.add("-sourcepath");
-        args.add(".");
-        compileTask.getOptions().setCompilerArgs(args);
     }
 
     private void readAndApplyJson(File file, String depConfig, String nativeConfig, Logger log)
-- 
2.13.6

