From e00b13e31dde9e889a177b905db0f384f20eaf05 Mon Sep 17 00:00:00 2001
From: Jonas Herzig <me@johni0702.de>
Date: Sat, 26 Aug 2017 17:29:13 +0200
Subject: [PATCH 2/2] Configure gradle to build reproducible archives

---
 build.gradle | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/build.gradle b/build.gradle
index b6c63bb..da0fdb9 100644
--- a/build.gradle
+++ b/build.gradle
@@ -16,6 +16,13 @@ jar {
     manifest.attributes('Main-Class': 'de.oceanlabs.mcp.mcinjector.MCInjector')
 }
 
+tasks.withType(AbstractArchiveTask) {
+    preserveFileTimestamps = false
+    reproducibleFileOrder = true
+    dirMode = 0775
+    fileMode = 0664
+}
+
 dependencies {
     compile 'org.ow2.asm:asm-debug-all:5.0.4'
     compile 'net.sf.jopt-simple:jopt-simple:4.5'
-- 
2.5.5

