From f1130f071e67235b27e2bca95584dec25a8dec37 Mon Sep 17 00:00:00 2001
From: Jonas Herzig <me@johni0702.de>
Date: Mon, 7 Aug 2017 20:04:56 +0200
Subject: [PATCH 2/3] Remove all buildscript dependencies

---
 build.gradle | 91 ------------------------------------------------------------
 1 file changed, 91 deletions(-)

diff --git a/build.gradle b/build.gradle
index c34a8a2..40169df 100644
--- a/build.gradle
+++ b/build.gradle
@@ -1,22 +1,6 @@
-buildscript {
-    repositories {
-        maven {
-            url "https://plugins.gradle.org/m2/"
-        }
-    }
-    dependencies {
-        classpath 'nl.javadude.gradle.plugins:license-gradle-plugin:0.11.0'
-        classpath "com.gradle.publish:plugin-publish-plugin:0.9.1"
-    }
-}
-
 apply plugin: 'groovy'
-apply plugin: 'license'
 apply plugin: 'maven'
 apply plugin: 'eclipse'
-apply plugin: "com.gradle.plugin-publish"
-
-defaultTasks 'licenseFormat', 'build'
 
 group = 'org.spongepowered'
 version = '0.4-SNAPSHOT'
@@ -43,22 +27,6 @@ processResources {
    from 'LICENSE.txt'
 }
 
-license {
-    ext {
-        name = project.name
-        organization = project.organization
-        url = project.url
-    }
-    include '**/*.groovy'
-    header file("HEADER.txt")
-    sourceSets = project.sourceSets
-    ignoreFailures false
-    strictCheck true
-    mapping {
-        java = 'SLASHSTAR_STYLE'
-    }
-}
-
 jar.manifest.mainAttributes (
     "Built-By": System.properties['user.name'],
     "Created-By": System.properties['java.vm.version'] + " (" + System.properties['java.vm.vendor'] + ")",
@@ -81,62 +49,3 @@ artifacts {
     archives groovydocJar
 }
 
-pluginBundle {
-    website = 'http://www.gradle.org/'
-    vcsUrl = 'https://github.com/SpongePowered/MixinGradle'
-    description = 'Gradle plugin for SpongePowered Mixin'
-    tags = ['spongepowered', 'sponge', 'mixin']
-    plugins {
-        patcher {
-            id = 'org.spongepowered.mixin'
-            displayName = 'SpongePowered Mixin Gradle Plugin'
-        }
-    }
-}
-
-uploadArchives {
-    repositories {
-
-        mavenDeployer {
-            configuration = configurations.deployerJars
-
-            if (project.hasProperty("chRepo"))
-            {
-                repository(url: project.chRepo) {
-                    authentication(userName: project.chUsername, password: project.chPassword)
-                }
-            }
-
-            pom {
-                groupId = project.group
-                version = project.version
-                artifactId = project.archivesBaseName
-                project {
-                    name project.archivesBaseName
-                    packaging 'jar'
-                    description 'MixinGradle'
-                    url 'http://www.spongepowered.org/'
-
-                    scm {
-                        url 'https://github.com/SpongePowered/Mixin'
-                        connection 'scm:git:git://github.com/SpongePowered/Mixin.git'
-                        developerConnection 'scm:git:git@github.com:SpongePowered/Mixin.git'
-                    }
-
-                    issueManagement {
-                        system 'GitHub Issues'
-                        url 'https://github.com/SpongePowered/MixinGradle/issues'
-                    }
-
-                    licenses {
-                        license {
-                            name 'MIT license'
-                            url 'http://opensource.org/licenses/MIT'
-                            distribution 'repo'
-                        }
-                    }
-                }
-            }
-        }
-    }
-}
-- 
2.5.5

