From 6b7ad7fd8c22384e62585a962e1bd74d6d003582 Mon Sep 17 00:00:00 2001
From: Jonas Herzig <me@johni0702.de>
Date: Sat, 2 Sep 2017 15:10:35 +0200
Subject: [PATCH] Disable license plugin

---
 build.gradle | 18 ------------------
 1 file changed, 18 deletions(-)

diff --git a/build.gradle b/build.gradle
index 60ad57d..57aa5e4 100644
--- a/build.gradle
+++ b/build.gradle
@@ -5,13 +5,11 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'nl.javadude.gradle.plugins:license-gradle-plugin:0.11.0'
         classpath "com.gradle.publish:plugin-publish-plugin:0.9.1"
     }
 }
 
 apply plugin: 'groovy'
-apply plugin: 'license'
 apply plugin: 'maven'
 apply plugin: 'eclipse'
 apply plugin: "com.gradle.plugin-publish"
@@ -42,22 +40,6 @@ processResources {
    from 'LICENSE.txt'
 }
 
-license {
-    ext {
-        name = project.name
-        organization = project.organization
-        url = project.url
-    }
-    include '**/*.groovy'
-    header file("HEADER.txt")
-    sourceSets = project.sourceSets
-    ignoreFailures false
-    strictCheck true
-    mapping {
-        java = 'SLASHSTAR_STYLE'
-    }
-}
-
 jar.manifest.mainAttributes (
     "Built-By": 'ReplayMod reprod',
     "Implementation-Title": name,
-- 
2.9.5

